packages/swapper/src/swappers/BebopSwapper/getBebopTradeQuote/getBebopTradeQuote.ts

```diff
@@ -115,13 +116,12 @@ export async function getBebopTradeQuote(
 
   try {
     const adapter = assertGetEvmChainAdapter(chainId)
-    const { average } = await adapter.getGasFeeData()
+    const { fast } = await adapter.getGasFeeData()
 
-    // Convert gas limit from hex to decimal string (Bebop returns hex values)
```

Comment:
Couldn't actually get hex values here despite trying many quotes - updated types to match what we're seeing here in responses

<img width="335" height="137" alt="image" src="https://github.com/user-attachments/assets/e7892dce-9ad8-4fdf-83fc-70a49975ad4b" />